### DISCUSSION TITLE
es/models/yolo12/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#19338] [Docs]
# es/models/yolo12/

Descubra YOLO12, con una innovadora arquitectura centrada en la atención para una detección de objetos de última generación con una precisión y eficacia inigualables.

https://docs.ultralytics.com/es/models/yolo12/

**Comment by IvanPeLe** [comment]
:)

**Reply by glenn-jocher** [reply]
@IvanPeLe hello! Thanks for reaching out and for the positive feedback. I appreciate you taking the time to comment. If you have any specific questions about using YOLO models or the Ultralytics framework, feel free to ask!

**Comment by UltralyticsAssistant** [comment]
👋 Hello, thank you for sharing your enthusiasm for YOLO 🚀! 

It seems you're referencing YOLO12 and its innovative architecture for next-generation object detection. While `YOLO8` is currently the latest officially supported release in our repository, we encourage you to visit the [Ultralytics Docs](https://docs.ultralytics.com/) for up-to-date information about supported models and their features.

If this is a ❓ Question or feature proposal, providing additional details and specifics can help us better understand your request. If you're experiencing a 🐛 Bug, please include a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) and relevant details to help us debug it more efficiently.

To ensure you're working with the latest advancements, upgrade to the most recent version of the `ultralytics` package and dependencies in a [**Python>=3.8**](https://www.python.org/) and [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) environment:

```bash
pip install -U ultralytics
```

YOLO can also be run in the following verified environments, pre-configured with all dependencies:

- **Notebooks** with free GPU access: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

For real-time conversations, you're welcome to join our community on [Discord](https://discord.com/invite/ultralytics) 🎧. You can also engage in deeper discussions at [Discourse](https://community.ultralytics.com/) or check out the latest topics on our [Subreddit](https://reddit.com/r/Ultralytics).

An Ultralytics engineer will review this discussion soon to provide further assistance 🙂. Stay tuned!

**Comment by deldanr** [comment]
When resuming a training session started with Yolov12, after the AMP check, the training session downloads a Yolov11 model to continue, which I assume is an error.

Attached is part of the log:
YOLOv12l summary: 488 layers, 26,389,875 parameters, 26,389,859 gradients, 89.4 GFLOPs

Transferred 1245/1245 items from pretrained weights
Freezing layer 'model.21.dfl.conv.weight'
AMP: Running Automatic Mixed Precision (AMP) checks...
Downloading https://github.com/ultralytics/assets/releases/download/v8.3.0/yolo11n.pt to 'yolo11n.pt'... <-- Here's the error
